### PR TITLE
Suspend qubole provider

### PR DESCRIPTION
**Pull request by potiuk**
Qubole has been acquired and seems that maintainers have left the project https://github.com/qubole/qds-sdk-py#where-are-the-maintainers- the package has been unmaintained for a long time and it's likely no-one uses it until someone steps up to maintain it, we suspend it

---
**^ Add meaningful description above**
Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Review by eladkal — APPROVED** [review]

**Review by Taragolis — APPROVED** [review]

**Comment by potiuk** [comment]
Interestingly enough @Taragolis boto is held still by something else as well:

https://github.com/apache/airflow/actions/runs/6012796736/job/16308910701#step:5:17858

(but all the mypy-boto3-stuff will get upgraded anyway to 1.28.36 as they all have been released today and our eager upgrade mechanism takes care about it)

**Comment by potiuk** [comment]
Right.. because for some reason we had qds-sdk as devel-only dependency as well :D

**Comment by potiuk** [comment]
Removed

**Comment by potiuk** [comment]
Way better: 


image

File: airflow/providers/SUSPENDING_AND_RESUMING_PROVIDERS.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ Suspending providers
 
 As of April 2023, we have the possibility to suspend individual providers, so that they are not holding
 back dependencies for Airflow and other providers. The process of suspending providers is described
-in `description of the process <https://github.com/apache/airflow/blob/main/README.md#suspending-releases-for-providers>`_
+in `description of the process <https://github.com/apache/airflow/blob/main/PROVIDERS.md>`_
```
Comment (jedcunningham) [inline airflow/providers/SUSPENDING_AND_RESUMING_PROVIDERS.rst]:
```suggestion
in `description of the process <https://github.com/apache/airflow/blob/main/PROVIDERS.rst#suspending-releases-for-providers>`_
```

**Review by jedcunningham — APPROVED** [review]

**Pull request merged**